docs/scripts/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```

Comment:
not sure if there's a better way to do this 😂

Reply:
I think we should be able to remove it once we upgrade nextjs to version 12 https://nextjs.org/docs/upgrading

> The minimum Node.js version has been bumped from 12.0.0 to 12.22.0 which is the first version of Node.js with native ES Modules support.

Reply:
We should already be using Node v16 which is the latest LTS version. https://github.com/aws-amplify/amplify-ui/blob/main/.nvmrc#L1

Reply:
Do you have any concern with using `type: module`, I think this will correctly tell Node to treat these files as ES Modules.

Reply:
Ohhhhh....It complained the version of **NodeJS**, I thought that was **NextJS** 😂 

Not sure why it complains the **NodeJS** version because ts-node is supposed to use the NodeJS version I use in the package, which is Node 16. 

As for `type: module`, no concerns, but I just think it doesn't look very clean to add the file here 😂